File: Java/Human.java
public class Human{
    
    private String NIK;
    private String name;
    private String gender;

    public Human(){
		this.NIK = "";
        this.name = "";
		this.gender = "";
	}

    public Human(String NIK, String name, String gender){
		this.NIK = NIK;
        this.name = name;
		this.gender = gender;
	}

    // Untuk NIK
	public String getNIK(){
		return this.NIK;
	}

	public void setNIK (String NIK){
		this.NIK = NIK;
	}

    // Untuk Nama
	public String getName(){
		return this.name;
	}

	public void setName (String name){
		this.name = name;
	}

    // Untuk Jenis Kelamin
	public String getGender(){
		return this.gender;
	}

	public void setGender (String gender){
		this.gender = gender;
	}
}

File: PHP/Human.php
<?php

class Human{

	// Atribut private dari Class Human
	private $NIK = "";
    private $name = "";
    private $gender = "";
	
	// Atribut public dari Class Human
	public function __construct($NIK = "", $name = "", $gender = ""){
		$this->NIK = $NIK;
		$this->name = $name;
        $this->gender = $gender;
	}

// Setter Dan Getter

    // Untuk NIK
	public function setNIK($NIK){
		$this->NIK = $NIK;
	}

	public function getNIK(){
		return $this->NIK;
	}

    // Untuk Name
    public function setName($name){
	    $this->name = $name;
	}

	public function getName(){
		return $this->name;
	}

    // Untuk Jenis Kelamin
	public function setGender($gender){
		$this->gender = $gender;
	}

	public function getGender(){
		return $this->gender;
	}
}

?>